libs/@local/hash-isomorphic-utils/src/blocks.ts

```diff
@@ -14,6 +14,8 @@ export type HashBlock = {
   meta: HashBlockMeta;
 };
 
+export type ComponentIdHashBlockMap = Record<string, HashBlock>;
```

Comment:
This used to be a type def in the frontend code, but is now used in the `save.ts` file, so i hoisted the type def and gave it a more precise name here.